src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Suppliers/Blocks/form.html.twig

```diff
@@ -34,77 +34,6 @@
   <div class="card-block row">
     <div class="card-text">
 
-      {% set invalidCatalogNameHint %}
-        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}
-      {% endset %}
-
-      {{ ps.form_group_row(supplierForm.name, {}, {
-        'label': 'Name'|trans({}, 'Admin.Global'),
-        'help': invalidCatalogNameHint
-      }) }}
-
-      {% set invalidGenericNameHint %}
-        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>={}' }}
-      {% endset %}
-
-      {% set descriptionHint %}
-        {% autoescape false %}
-          {{ 'Will appear in the list of suppliers.'|trans({}, 'Admin.Catalog.Help')~ invalidCatalogNameHint }}
-        {% endautoescape %}
-      {% endset %}
-
-      {{ ps.form_group_row(supplierForm.description, {}, {
-        'label': 'Description'|trans({}, 'Admin.Global'),
-        'help': descriptionHint
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.phone, {}, {
-        'label': 'Home phone'|trans({}, 'Admin.Global'),
-        'hint': 'Phone number for this supplier'|trans({}, 'Admin.Catalog.Help')
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.mobile_phone, {}, {
-        'label': 'Mobile phone'|trans({}, 'Admin.Global'),
-        'hint': 'Mobile phone number for this supplier.'|trans({}, 'Admin.Catalog.Help')
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.address, {}, {
-        'label': 'Address'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.address2, {}, {
-        'label': 'Address (2)'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.post_code, {}, {
-        'label': 'Zip/postal code'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.city, {}, {
-        'label': 'City'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.id_country, {'attr': {
-        'class': 'js-supplier-country-select',
-        'data-states-url': path('admin_country_states') }}, {
-        'label': 'Country'|trans({}, 'Admin.Global'),
-      }) }}
-
-      <div class="js-supplier-state{% if supplierForm.id_state.vars.choices is empty %} d-none{% endif %}">
-        {{ ps.form_group_row(supplierForm.id_state, {}, {
-          'label': 'State'|trans({}, 'Admin.Global'),
-        }) }}
-      </div>
-
-      {{ ps.form_group_row(supplierForm.dni, {}, {
-        'label': 'DNI'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(supplierForm.logo, {}, {
-        'label': 'Logo'|trans({}, 'Admin.Global'),
-        'help': 'Upload a supplier logo from your computer.'|trans({}, 'Admin.Catalog.Help')
-      }) }}
-
       {% if logoImage is defined and logoImage is not null %}
```

Comment:
Why do you change order for this element ?

Reply:
Yeah that was a mistake, moved it where it should be. Thanks!